CONTRIBUTING.md

```diff
@@ -91,21 +80,85 @@ To see test coverage, run `dotnet test` from a console window and you will see t
 
 Or, after running the tests, open the file `TestResults\Results\index.htm` in a browser.
 
-### Coding style
+## Pull Requests
 
-This project includes a [`.editorconfig`](https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/.editorconfig)
-file which is supported by all the IDEs/editor mentioned above.
-It works with the IDE/editor only and does not affect the actual build of the project.
+### How to Send Pull Requests
+
+Everyone is welcome to contribute code to `opentelemetry-dotnet` via GitHub
+pull requests (PRs).
+
+To create a new PR, fork the project in GitHub and clone the upstream repo:
+
+```sh
+$ git clone https://github.com/open-telemetry/opentelemetry-dotnet.git
+```
 
-This repository also includes a
-[_test_, _prod_ and _prod.loose_ _stylecop ruleset_](https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/build) files.
+Add your fork as an origin:
+
+```sh
+$ git remote add fork https://github.com/YOUR_GITHUB_USERNAME/opentelemetry-dotnet.git
+```
+
+Run tests:
+
+```sh
+dotnet test
+```
 
-These files are used to configure the _StyleCop.Analyzers_ which runs during build. Breaking the rules will result in a broken build.
+Check out a new branch, make modifications and push the branch to your fork:
 
-### Proposing changes
+```sh
+$ git checkout -b feature
+# edit files
+$ git commit
+$ git push fork feature
+```
+
+Open a pull request against the main `opentelemetry-dotnet` repo.
+
+### How to Receive Comments
+
+* If the PR is not ready for review, please put `[WIP]` in the title, tag it
+  as `work-in-progress`, or mark it as [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+* Make sure CLA is signed and CI is clear.
+
+### How to Get PRs Merged
+
+A PR is considered to be **ready to merge** when:
+* It has received approval from [Approvers](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver).
```

Comment:
I would mention that it didn't receive "Ask for change" from any approver or maintainer as well.

Reply:
This is covered in the next bullet point: "Major feedbacks are resolved."